go/test/fuzzing/oss_fuzz_build.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+# Copyright 2021 The Vitess Authors.
+# 
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# 
+#     http://www.apache.org/licenses/LICENSE-2.0
+# 
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+compile_go_fuzzer ./go/vt/sqlparser Fuzz fuzz
```

Comment:
Hmm... What do we do about this dependency? `compile_go_fuzzer` is not something most people have installed, right?

Reply:
compile_go_fuzzer is available when the file is run through OSS-fuzz.
The file is meant to be run only by OSS-fuzz's setup.